package.json

```diff
@@ -6,10 +6,7 @@
   "workspaces": {
     "packages": [
       "packages/*",
-      "examples/graphiql-webpack",
-      "examples/monaco-graphql-webpack",
-      "examples/monaco-graphql-react-vite",
-      "examples/cm6-graphql-parcel"
+      "examples/*"
```

Comment:
I guess it's fine to include all examples in workspace

Reply:
we removed the examples from linting a while ago, this slows things down and isn't very valuable

Reply:
here is `workspaces` field in `package.json`, do you want to completely remove all `examples` from here to avoid installing their dependencies?

We still can keep ESLint lint for examples folder, to avoid some mistakes